.ci/run-link-check-plugin.sh

```diff
@@ -3,19 +3,25 @@
 #This script is used at https://app.codeship.com/projects/124310
 #Run "firefox target/site/linkcheck.html" after completion to review html report
 
-set -e
-pwd
-uname -a
-mvn --version
-curl -I https://sourceforge.net/projects/checkstyle/
-mvn -e clean site -Dcheckstyle.ant.skip=true -DskipTests -DskipITs \
-   -Dpmd.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true
-echo "------------ grep of linkcheck.html--BEGIN"
-# "grep ... | cat" is required command is running in "set -e" mode and
-# grep could return exit code 1 if nothing is matching
-grep externalLink target/site/linkcheck.html | cat
-echo "------------ grep of linkcheck.html--END"
-RESULT=$(grep externalLink target/site/linkcheck.html | grep -v 'Read timed out' | wc -l)
-echo 'Exit code:'$RESULT
-if [[ $RESULT != 0 ]]; then false; fi
+# To skip build if minor files or other CI files area changed
+SKIP_FILES=".github|appveyor.yml|circle.yml|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|\
+            LICENSE.apache20|README.md|release.sh|RIGHTS.antlr|shippable.yml|wercker.yml"
```

Comment:
please add this code to https://github.com/checkstyle/checkstyle/blob/master/codeship-steps.yml 
we need to skip only when we are in CI mode. This shell file can be executed at local and it should not skip anything.

Reply:
Alright, got it 👍

Reply:
@romani Done.